layer.py

```diff
@@ -241,6 +241,7 @@ def get_attachment_type_by_int_value(self, value: int) -> AttachmentType:
 
     def attachment_naming(self, field_name) -> str:
         attachment_type = self.get_attachment_field_type(field_name)
+        assert attachment_type is not None
```

Comment:
@m-kuhn , is that supposed to be in your commit?

Reply:
get_attachment_field_type can return `None` but `ATTACHMENT_EXPRESSIONS` (next line below here) doesn't have a `None` value. If we ever end up in here with `None`, we have a problem to investigate.

Reply:
@m-kuhn , ok, just checking this was not an accidental leftover from some local testing :)

Reply:
Not at all, it's `mypy` hinting potential problems (and silencing these by adding asserts that mean "trust us, we know what we are doing"